src/client/fuse_ll.cc

```diff
@@ -980,6 +1007,16 @@ static void do_init(void *data, fuse_conn_info *conn)
   CephFuse::Handle *cfuse = (CephFuse::Handle *)data;
   Client *client = cfuse->client;
 
+#ifdef CEPH_HAVE_SPLICE
+  auto fuse_read_splice = cfuse->client->cct->_conf.get_val<bool>(
+    "fuse_read_splice");
+
+  if (fuse_read_splice) {
+      conn->want |= FUSE_CAP_SPLICE_READ | FUSE_CAP_SPLICE_WRITE;
+      conn->want |= FUSE_CAP_SPLICE_MOVE;
```

Comment:
You mentioned in an earlier comment that splice write wasn't helpful, but you're setting the flag for it here. Is that deliberate?

Reply:
Here the `FUSE_CAP_SPLICE_WRITE` will be used by splice read when writing the readed data and fuse header info to the pipe fd in libfuse. And the FUSE_CAP_SPLICE_READ will be used in libfuse daemon thread loop when receiving data and fuse header info from one pipe from kernel.

Actually for the splice write it is partially implemented when we enable and using the `FUSE_CAP_SPLICE_READ `.  If the `fuse_read_splice` option make sense we may just rename it to `fuse_splice` ?

Reply:
Maybe I was not correctly comment the splice write, there is no need to do extra work in ceph-fuse code just like the splice read :-)